src/main/java/sx/blah/discord/handle/impl/events/user/StatusChangeEvent.java

```diff
@@ -5,7 +5,9 @@
 
 /**
  * This event is dispatched when a user's status changes.
+ * @deprecated Use {@link PresenceUpdateEvent}
  */
+@Deprecated
 public class StatusChangeEvent extends UserEvent {
```

Comment:
You could delete this event since its new to the dev branch, you should also link to the PresenceUpdateEvent in the old event class too.